ansible/README.md

```diff
@@ -0,0 +1,31 @@
+# Ansible deploy script
```

Comment:
Style note: please put all sentences on separate lines; don't break sentences midline unless there's a clear separator between 2 subclauses (such as ", or", which is a good breaking point).

This makes the markdown easier to edit in the future.